game/test_streak.py

```
@@ -0,0 +1,98 @@
+from datetime import timedelta
+from unittest import mock
+
+from django.test import TestCase
+from django.utils import timezone
+from django.contrib.auth.models import User
+from game.models import UserProgress
+from game.views import record_game_result
+
+class StreakCounterTest(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username='testuser', password='password')
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to argument: "password"

(S106)